### PR TITLE
Convert feature test to system tests

### LINKED ISSUE #388: Convert feature tests to system tests
I have done this [once before](https://github.com/umts/umaps-rails/pull/1157/files) and I suggest looking through my previous work if undertaking this. There were a few things I learned the hard way that you don't have to!

### PR DESCRIPTION
**Pull request by Tanvish**
Should close #388 
- Mostly included all the information in the commit messages but there aren't any major changes per say.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -54,7 +56,10 @@ group :test do
   gem 'rails-controller-testing'
   gem 'rspec-html-matchers'
   gem 'rspec-rails'
+  gem 'selenium-webdriver'
   gem 'simplecov'
   gem 'timecop'
   gem 'umts-custom-matchers'
+  gem 'vcr'
```
Comment (Anbranin) [inline Gemfile:63]: Do we need this gem? Where are we using it?

**Review by Anbranin — CHANGES_REQUESTED** [review]

File: Gemfile [hunk 2]
```diff
@@ -54,7 +56,10 @@ group :test do
   gem 'rails-controller-testing'
   gem 'rspec-html-matchers'
   gem 'rspec-rails'
+  gem 'selenium-webdriver'
```
[the remark this answers is not in the document]
Comment (werebus) [reply Gemfile:59]: Karin should have specified, but you should have only `webdrivers` in here.

`webdrivers` depends on `selenium-webdriver`, but also does some helpful, "download the correct version and tell selenium to use it" work for us.

[the remark this answers is not in the document]
Comment (werebus) [reply Gemfile:59]: Also, don't forget to bundle and commit the `Gemfile.lock` when changing the `Gemfile`.

**Review by werebus — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 2 above)
[the remark this answers is not in the document]
Comment (Tanvish) [reply Gemfile:59]: Yeah, I wasn't sure but good to know and I forgot to bundle sorry!

**Review by werebus — APPROVED** [review]

**Review by Anbranin — APPROVED** [review]

**Pull request merged**